### ISSUE TITLE
Train Loss Problem

### ISSUE THREAD

**Issue by DataXujing** [princeton-vl/CornerNet-Lite#130]
When I train my own data, used CornerNet_saccade model, but the loss do not decrease, in my training process. I do not known why, should I need a pre_trained model?

**Comment by float4189** [comment]
Hello! I also encountered the same problem,Did you solve it?

**Comment by DataXujing** [comment]
> Hello! I also encountered the same problem,Did you solve it?

I do not Known how to treat it. I think, Pre-trained model could solve this problem.

**Comment by float4189** [comment]
OK,！thank you very much, which model are you using now?
